src/cardstack/al-cardstack-view.ts

```diff
@@ -243,6 +278,11 @@ export abstract class AlCardstackView< EntityType=any,
      */
     public abstract async fetchData( initialLoad:boolean ):Promise<EntityType[]>;
 
+    /**
+     *  It call every time the something happend with the list
+     */
+    public abstract cardsChange():void;
```

Comment:
Since this becomes a major part of the cardstack's API, I would

    - Make this method name obey the triggered event naming convention -- e.g., `onCardsChanged`.
    - Consider making it optional?  Not all implementations will care about this.

Reply:
thanks Kevin